### PR TITLE
Lantern 232

### PR DESCRIPTION
**Pull request by mrnosal**
Pull requests into this repository require the following checks to be completed by the submitter and reviewers.

**Submitter:**
- [x] Administrative tasks are complete.
  - This pull request describes why these changes were made.
  - The JIRA ticket links to this PR.
  - The JIRA ticket for this PR is: https://oncprojectracking.healthit.gov/support/browse/LANTERN-232
- [x] Tests are complete.
  - Tests are included and test edge cases.
  - Tests have been run locally and pass.
- [x] Code review has been performed.
  - Code diff has been done and checked to ensure only expected code is being committed.
  - A linter for the language has been run.

**Primary Reviewer:**

- [x] Code works.
  - Code has been run locally and the basic functionality affected by the PR has been checked.
  - Edge cases have been checked to ensure behavior is as expected.
- [x] Code review has been performed.
  - Code is maintainable, reusable, efficient, and correct.
  - Code accomplishes the tasks purpose.
  - Code follows style guidance appropriate for the language, including passing linter checks.
  - Code is well commented (which does not mean verbosely commented).
- [x] Tests are complete.
  - Tests are included and test edge cases.
  - Tests have been run locally and pass.

**Secondary Reviewer:**

- [ ] Code works.
  - Code has been run locally and the basic functionality affected by the PR has been checked.
  - Edge cases have been checked to ensure behavior is as expected.
- [x] Code review has been performed.
  - Code accomplishes the tasks purpose.
- [x] Tests are complete.
  - Tests have been run locally and pass.

**Merger:**

Prior to merging, ensure that all checks have been completed. Delete the branch after the merge
is complete.

Create a new branch `<branch name>_gomodupdate`. Run `make update_mods` to update the `go.mod`
and `go.sum` files to point to master. Create a PR. Require one sanity check reviewer.

### CODE REVIEW

File: shinydashboard/lantern/functions/endpoints.R [hunk 1]
```diff
@@ -197,4 +200,159 @@ get_avg_response_time <- function(db_connection) {
...
+    tidyr::replace_na(list(fhir_version = "Unknown"))
+}
+
+# Return a summary table of information about endpoint security statements
+get_well_known_endpoint_counts <- function(db_connection) {
+  res <- tribble(
+    ~Status, ~Endpoints,
+    "Total Indexed Endpoints", as.integer(app_data$fhir_endpoint_totals$all_endpoints),
+    "Endpoints with successful response (HTTP 200)", as.integer(app_data$response_tally$http_200),
+    "Well Known URI Endpoints with succesful response", get_well_known_endpoints_count(db_connection),
```
Comment (emichaud998) [inline shinydashboard/lantern/functions/endpoints.R:323]: "succesful" spelling typo

File: shinydashboard/lantern/functions/endpoints.R [hunk 2]
```diff
@@ -197,4 +200,159 @@ get_avg_response_time <- function(db_connection) {
...
+  ) %>% pull(count)
+}
+
+# Return a summary table of information about endpoint security statements
+get_endpoint_security_counts <- function(db_connection) {
+  res <- tribble(
+    ~Status, ~Endpoints,
+    "Total Indexed Endpoints",as.integer(app_data$fhir_endpoint_totals$all_endpoints),
+    "Endpoints with successful response (HTTP 200)",as.integer(app_data$response_tally$http_200),
+    "Endpoints with unsuccesful response",as.integer(app_data$response_tally$http_non200),
```
Comment (emichaud998) [inline shinydashboard/lantern/functions/endpoints.R:354]: Another small spelling typo with "unsuccesful"

File: shinydashboard/lantern/functions/endpoints.R (hunk 1 above)
Comment (mayerm94) [inline shinydashboard/lantern/functions/endpoints.R:323]: Should also specify (HTTP 200) int he same way that was done for successful endpoint response in the line above

**Comment by mrnosal** [comment]
> Besides two minor typos, this looks great to me! I just had one other small suggestion, it is up to you and Matt if you would actually like to change it or not, but I was thinking it might look better to move the fhir version and vendor drop downs above the table they affect, rather than above the lists that do not change based on the filter, just so it is clear what the filter is for

The filters are added outside the module, so it will require refactoring to move them lower in the UI. Also, changing the summary tables to reflect the filters will require additional refactoring.

File: shinydashboard/lantern/functions/endpoints.R (hunk 1 above)
Comment (mrnosal) [reply shinydashboard/lantern/functions/endpoints.R:323]: Will update table label to match

**Pull request merged**